Userland/Utilities/xxd.cpp

```diff
@@ -261,9 +330,16 @@ ErrorOr<int> serenity_main(Main::Arguments args)
         }
     }
 
-    // TODO: colorize output
+    bool colorize_output = false;
     if (!colorize_output_option.is_null()) {
-        warnln("Colorizing output is not supported");
+        if (colorize_output_option == "always" || colorize_output_option == "auto") {
+            colorize_output = true;
+        } else if (colorize_output_option == "never") {
+            colorize_output = false;
+        } else {
+            warnln("Unknown value '{}' for -R", colorize_output_option);
```

Comment:
Maybe add `, should be one of 'always', 'auto', 'never'` to make the error message more helpful.